<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1331</width>
    <height>873</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>20</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>10</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어 Bold</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>선택한 색상</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777209</width>
     <height>16777212</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777209</width>
     <height>16777212</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>90</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777209</width>
     <height>16777212</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>90</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777209</width>
     <height>16777212</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>400</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777209</width>
     <height>16777212</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>400</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777209</width>
     <height>16777212</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>400</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777209</width>
     <height>16777212</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>400</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777209</width>
     <height>16777212</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="HOME">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어 Bold</family>
     <pointsize>24</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>돌아가기</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>720</y>
     <width>811</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어 Bold</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>원하는 옷을 선택해주세요</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
